From 2cd7621d4ca4a3f78dbfb6b15779cbf57770670a Mon Sep 17 00:00:00 2001
From: Stuart McLaren <stuart.mclaren@hp.com>
Date: Sun, 12 Mar 2017 21:21:43 +0000
Subject: [PATCH] Allow configure to run on netbsd

NetBSD's shell (/bin/sh) doesn't support '+='. Removing '+=' allows
configure to run and the library to compile.

This matches configure.ac in boothj5/profanity which also doesn't
use '+='.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6d19f2e..34a5fe4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -18,11 +18,11 @@ AS_CASE([$host_os],
                    [PLATFORM="nix"])

 PKG_CHECK_MODULES([expat], [expat >= 2.0.0],
-                  [PC_REQUIRES+=(expat)],
+                  [PC_REQUIRES="expat ${PC_REQUIRES}"],
                   [AC_CHECK_HEADER([expat.h],
                                    [
                                     expat_LIBS="-lexpat"
-                                    PC_LIBS+=($expat_LIBS)
+                                    PC_LIBS="${expat_LIBS} ${PC_LIBS}"
                                    ],
                                    [AC_MSG_ERROR([expat not found; expat required.])]
                                   )
